websites/H/HackerRank/presence.ts

```diff
@@ -0,0 +1,51 @@
+const presence = new Presence({
+  clientId: "655534149871992845"
+});
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: Math.floor(Date.now() / 1000)
+  },
```

Comment:
you should probs set your Date.now outside of the updatadata event

Reply:
why 🤔

Reply:
Because you will set a new time now every updateData event, instead of just keep setting the same.